gtda/plotting/diagrams.py

```diff
@@ -206,3 +322,34 @@ def plot_landscapes(landscapes, homology_dimensions=None, samplings=None):
                                        name="layer {}".format(layer + 1)))
 
         fig.show()
+
+
+def plot_persistence_image(images, homology_dimension=None, samplings=None):
```

Comment:
If we go for plural in `plot_betti_curves` we should also go for plural here and in the heat kernel case.

Reply:
In general I guess we would like all functions in plotting (maybe betti surfaces are the only exceptions) to plot a single output for now, so should we always go for singular?

Reply:
The nuance is that we plot all (specified) dimensions (potentially more than 1) for betti curves, on the plot.
For images, we will only plot one.
Hence the difference. I agree it is not ideal, i can change the names or/and the behaviorof those functions.